components/config-mgmt-service/backend/elastic/runs.go

```diff
@@ -158,6 +158,56 @@ func (es Backend) GetRuns(nodeID string, page int, perPage int, filters map[stri
 	return runs, nil
 }
 
+func (es Backend) GetRunsPageByCurser(ctx context.Context, nodeID string, start time.Time,
```

Comment:
This function allows walking through all the runs of a node with no 10,000 item limit.